tests/test_boxes.py

```diff
@@ -155,24 +200,31 @@ def test_label_invariance(self, label0: int, label1: int, shuffle_inds: List[int
                          [10., 10., 11, 11]])   # iou = 1 (truth 1) should be marked positive
 
         # xyxy format
-        truth = np.array([[-.5, -.5, .5, .5],
-                          [10., 10., 11, 11]])
+        truth = np.array([[-0.5, -0.5, 0.5, 0.5], [10.0, 10.0, 11, 11]])
```

Comment:
I prefer this on two lines. Will black leave it alone if we just change the indent?

``` python
truth = np.array([
    [-0.5, -0.5, 0.5, 0.5],
    [10.0, 10.0, 11, 11],
])
```